Task-2-Days-of-week/EasyLVL.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_2_Days_of_week
+{
+    class EasyLVL
+    {
+        public enum DaysOfWeek
+        {
+            monday = -999,
+            tuesday,
```

Comment:
```suggestion
            Tuesday,
```
enum values usually start from the capital later